.github/workflows/bump_private_action_version.yml

```diff
@@ -7,9 +7,10 @@ on:
 jobs:
     call-version-getter-shared-workflow:
         permissions:
-            contents: read
+            contents: write
```

Comment:
this might also need pull-request: write if it works the same as the synthetics one